definitions/variable/industry/tag_industry_sectors.yaml

```
@@ -0,0 +1,26 @@
+# Industry sectors as used in IEA World Energy Balances (2022 Edition)
+- Industry Sector:
+    - Mining:
+          description: industry sectors categorized as ISIC Rev. 4 Divisions 07 and 08 and Group 099
```

Suggest to still repeat the actual sector name in the description
```suggestion
        description: industry sectors categorized as ISIC Rev. 4 Divisions 07 and 08 and Group 099
```